### PR TITLE
Update readme to fit Solgate

### PR DESCRIPTION
**Pull request by tumido**
## Related Issues and Dependencies

#7 

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

A documentation change - updates the Readme to better fit `solgate` instead of `dh-argo-workflows`.

### CODE REVIEW

**Comment by 4n4nd** [comment]
@tumido  can you add a bit of a description for the changes in `s3.py`?

**Comment by tumido** [comment]
@4n4nd  Yeah, sure! :+1: I'll probably rather scrape it away from this PR and issue a more robust fix tomorrow. :slightly_smiling_face:

File: README.md [hunk 1]
```diff
@@ -1,181 +1,166 @@
...
-```bash
-$ ansible-playbook playbook.yaml \
-    --ask-vault-pass
-    -e kubeconfig=$HOME/.kube/config \
-    -e target_env=prod
+```ini
+[destination_some_fancy_name]
+aws_access_key_id     = KEY_ID
+aws_secret_access_key = SECRET
+base_path             = DH-PLAYPEN/storage/input       ; at least the bucket name is required, sub path within this bucket is optional
```
[the remark this answers is not in the document]
Comment (tumido) [reply README.md:57]: fixed :+1:

**Comment by tumido** [comment]
> LGTM 
> Do you think it would be a good idea to add an example `Configuration` file?

@4n4nd good idea. :+1: I've added an issue for that and will solve it in another PR. :slightly_smiling_face: #11

**Pull request merged**